### ISSUE TITLE
第4章的anchor_target_layer.py中代码格式似乎有问题

### ISSUE THREAD

**Issue by Eggache666** [dongdonghy/Detection-PyTorch-Notebook#18]
代码位置：chapter4/faster-rcnn-pytorch/lib/model/rpn/anchor_target_layer.py 的第166行

num_examples = torch.sum(labels[i] >= 0)
请问方括号里的 i 是在哪里定义的？是上面for语句里面的吗？是不是格式化错了？能否调整下，谢谢！
